helm/prometheus-operator/templates/servicemonitor-configmap.yaml

```diff
@@ -25,5 +25,5 @@ data:
           matchNames:
             - {{ .Release.Namespace | quote }}
         endpoints:
-        - port: web
-          interval: 30s
\ No newline at end of file
+        - port: html
```

Comment:
that would be `http` not `html`

Reply:
Of course sorry, this is what I meant ...